game/src/game_core/resources.rs

```diff
@@ -6,12 +6,22 @@ use std::ops;
 /// This struct holds data for resources
 /// This is used to describe the current state and change rate of the player's resources.
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
-pub struct Resources<T: PartialOrd> {
+pub struct Resources<T> {
     pub(crate) oxygen: T,
     pub(crate) energy: T,
     pub(crate) life: T,
 }
 
+impl Default for Resources<i16> {
+    fn default() -> Self {
+        Self {
+            oxygen: 0,
+            energy: 0,
+            life: 0,
+        }
```

Comment:
Sollten Standartwerte nicht auf MAX sein? zumindest nicht 0 --> Todesscreen idk

Reply:
Approved und die default implementierung is doch hier nicht nötig

Reply:
Ja würde ich auch rausnehmen. Solange wir die default Funktion nicht mehrfach nutzen ist die wenig sinnvoll

Reply:
Also ich würde halt immer dann die default nutzen wenn ich al maschien eine mesage senden möchte um dann daten zusetzen welche ich als mesage senden möchte 
bzw beim erstellen brauch ihc auch nen 000